### PR TITLE
Update doc and thrown message

### PR DESCRIPTION
**Pull request by kbruniec**
### Details:
 - Multiply, Add and Subtract layers auto broadcasting is only supported for inputs which are constant.
 - Add documentation
 - Extend thrown message

### Tickets:
 - 32541, 32542

### CODE REVIEW

**Review by nkogteva — APPROVED** [review]

**Review by dorloff — APPROVED** [review]

File: docs/IE_DG/supported_plugins/GNA.md [hunk 1]
```diff
@@ -230,6 +230,7 @@ Limitations include:
 - The maximum number of filters is 65532 for GNA 2.0 and 8192 for GNA 3.0.
 - Permute layer support is limited to the cases where no data reordering is needed or when reordering is happening for two dimensions, at least one of which is not greater than 8.
 - Splits and concatenations are supported for continuous portions of memory (e.g., split of 1,2,3,4 to 1,1,3,4 and 1,1,3,4 or concats of 1,2,3,4 and 1,2,3,5 to 2,2,3,4).
+- For Multiply, Add and Subtract layers auto broadcasting is only supported for inputs which are constant.
```
Comment (andrew-zaytsev) [inline docs/IE_DG/supported_plugins/GNA.md:233]: --> For Multiply, Add and Subtract layers, auto broadcasting is only supported for constant inputs.

Comment (dorloff) [inline docs/IE_DG/supported_plugins/GNA.md:233]: @andrew-zaytsev suggests to replace with "constant inputs".

File: src/plugins/intel_gna/gna_graph_compiler.cpp [hunk 2]
```diff
@@ -1242,7 +1242,9 @@ void GNAGraphCompiler::EltwisePrimitive(InferenceEngine::CNNLayerPtr layer) {
     }
 
     if (in_4b_total_size != in_2b_total_size) {
-        THROW_GNA_LAYER_EXCEPTION(layer) << " Inputs size mismatch " << in_4b_total_size << " != " << in_2b_total_size;
+        THROW_GNA_LAYER_EXCEPTION(layer) << " Inputs size mismatch "
+            << "(note: For Multiply, Add and Subtract layers auto broadcasting is only supported for inputs which are constant) "
```
Comment (dorloff) [inline src/plugins/intel_gna/gna_graph_compiler.cpp:1246]: @andrew-zaytsev suggests to replace with "constant inputs".

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

File: docs/IE_DG/supported_plugins/GNA.md (hunk 1 above)
Comment (kbruniec) [reply docs/IE_DG/supported_plugins/GNA.md]: done

File: src/plugins/intel_gna/gna_graph_compiler.cpp (hunk 2 above)
Comment (kbruniec) [reply src/plugins/intel_gna/gna_graph_compiler.cpp]: done

File: docs/IE_DG/supported_plugins/GNA.md (hunk 1 above)
Comment (kbruniec) [reply docs/IE_DG/supported_plugins/GNA.md]: done

**Review by andrew-zaytsev — APPROVED** [review]

**Pull request merged**